contracts/SBCDepositContract.sol

```diff
@@ -299,6 +299,17 @@ contract SBCDepositContract is
         bool _unwrapToGNO,
         uint256 gasLimit
     ) internal returns (bool success) {
+        // Skip withdrawal that burns tokens to avoid a revert condition
+        // https://github.com/OpenZeppelin/openzeppelin-contracts/blob/dad73159df3d3053c72b5e430fa8164330f18068/contracts/token/ERC20/ERC20.sol#L278
+        if (_receiver == address(0)) {
```

Comment:
Why this is necessary? Burn of mGNO token will be done with `account` equal to address of withdrawals contract.

Reply:
- Please see https://github.com/gnosischain/deposit-contract/pull/30